modules/ROOT/pages/mmp-concept.adoc

```diff
@@ -158,7 +158,10 @@ This feature is available in plugin version 3.2.5 and later.
 [[configure-shared-libraries]]
 == Configure Shared Libraries
 
-Configure shared libraries when you need to make an application's dependency visible to your application's connectors. To configure a dependency as a shared library, add the following elements in your Mule Maven plugin configuration inside your project's `pom.xml` file:
+Shared libraries enable you to configure a dependency in your application and make this dependency accessible to all the connectors in your application. +
+See xref:mule-runtime::about-classloading-isolation.adoc#shared-libraries[Shared Libraries] for an in-depth explanation about this feature.
+
+To configure a dependency as a shared library, add the following elements in your Mule Maven plugin configuration inside your project's `pom.xml` file:
```

Comment:
I got a little confused here by "on" and "inside"
Are we asking the audience to add the elements **to** the plugin config or to the POM?